extractor/src/main/resources/config-template.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name = "Checker">
+    <property name="charset" value="UTF-8"/>
+
+    <!-- do not change severity to 'error', as that will hide errors caused by exceptions -->
+    <property name="severity" value="warning"/>
+
+    <!-- haltOnException is required for exception fixes and reporting of all exceptions -->
+    <property name="haltOnException" value="false"/>
+
+    <!-- BeforeExecutionFileFilters is required for sources of java9 -->
+    <module name="BeforeExecutionExclusionFileFilter">
+        <property name="fileNamePattern" value="module\-info\.java$" />
+    </module>
+
+    <module name="TreeWalker">
+        <module name="$MODULE_NAME">
+            $PROPERTIES_PLACEHOLDER
```

Comment:
@relentless-pursuit , I don't see it, please push

Reply:
> @relentless-pursuit , I don't see it, please push

I did add a comment to my file. Can you confirm? I can push again though.

Reply:
It will be better to have template lik:
```
<module name="TreeWalker">
        <!-- treewalker_module_PLACEHOLDER -->
```

So the same template will be used for single module and all-in-one

Reply:
if i understood it right, you wanted me to add html kind comments. i have done that. please let me know.

Reply:
> Please also rename file to put in name "treewalker" as such config will be used only for Checks with patent Treewalker. There are bunch of modules that have Checker as parent, so treewalker is not required at all and should not be in config at all. Example of module of Checker is LineLength



> Please also rename file to put in name "treewalker" as such config will be used only for Checks with patent Treewalker. There are bunch of modules that have Checker as parent, so treewalker is not required at all and should not be in config at all. Example of module of Checker is LineLength



> Please also rename file to put in name "treewalker" as such config will be used only for Checks with patent Treewalker. There are bunch of modules that have Checker as parent, so treewalker is not required at all and should not be in config at all. Example of module of Checker is LineLength

done

Reply:
Content should be:
```
<module name="TreeWalker">
        <!-- treewalker_module_PLACEHOLDER -->
        
```

You should search for `        <!-- treewalker_module_PLACEHOLDER -->` and replace it with generated content